app/lib/status.py

```diff
@@ -34,6 +34,28 @@ def __init__(self, message='Bad request', details=None, result=None,
                                          result=result, info=info)
 
 
+class Unauthorized(TodoException):
+    """Exception where unauthorized request is made.
```

Comment:
Could you rephrase this to be `Exception raised when an unauthorized request is made.`